src/dawn/CodeGen/CodeGen.cpp

```diff
@@ -67,8 +67,8 @@ void CodeGen::addTmpStorageInit_wrapper(MemberFunction& ctr,
                                         const std::vector<std::string>& tempFields) const {
   if(!(tempFields.empty())) {
     auto verticalExtent = getVerticalTmpHaloSizeForMultipleStencils(stencils);
-    ctr.addInit(bigWrapperMetadata_ + "(dom.isize(), dom.jsize(), dom.ksize() + 2*" +
-                std::to_string(verticalExtent) + ")");
+    ctr.addInit(bigWrapperMetadata_ + "(dom.isize(), dom.jsize(), dom.ksize() /*+ 2 *" +
```

Comment:
are you sure about commenting this 2* verticalExtent out?

Reply:
yes, needs to be as is right now. We can discuss the GT issue tomorrow?

Reply:
I understand you need to add +1, but are you sure we need to remove the vertical extent?

Reply:
at some point this led to very cryptic GT errors (shared pointer to storage is not the stored element). Is this resolved now? If so, we can uncomment and use the proper computation here